v0/destinations/ga/transform.js

```diff
@@ -56,23 +52,42 @@ function responseBuilderSimple(
   mappingJson,
   destination
 ) {
-  const rawPayload = {
-    v: "1",
-    t: hitType,
-    tid: destination.Config.trackingID,
-    ds: message.channel,
-    an: message.context.app.name,
-    av: message.context.app.version,
-    aiid: message.context.app.namespace
-  };
-  if (destination.Config.doubleClick) {
+  const {
+    doubleClick,
+    anonymizeIp,
+    enhancedLinkAttribution,
+    dimensions,
+    metrics,
+    contentGroupings
+  } = destination.Config;
+  var rawPayload;
+  if (message.context.app) {
+    rawPayload = {
+      v: "1",
+      t: hitType,
+      tid: destination.Config.trackingID,
```

Comment:
we can add this under desctructure as well?

Reply:
done